src/testup/editor.rb

```diff
@@ -74,8 +74,9 @@ def self.open_file(file, line = 0)
     arguments = self.arguments
     arguments.gsub!('{FILE}', file)
     arguments.gsub!('{LINE}', line)
-    command = %["#{editor}" #{arguments}]
-    system(command)
+    # Do not occupy the current process
+    arr = arguments.gsub("\"","").split(" ")
```

Comment:
What does this line do? Remove quotes?

Reply:
yes, the quotes are useless.
when I use notepad++, the "\"file_path -nrow\"" will make a mistake.
Kernel.spawn(editor,"\"file_path","-nrow\"").

Reply:
Is the quotes treated differently by `spawn` than `system`?  I'm pretty sure they where needed before. Maybe its editor specific....

Reply:
Because of the blank space. 
The sublime text is {file}:{line}, but the notepad++ is {file}_-n{line}.
'_' stands for blank space.
After splitting, the quotes are not in a string.

Reply:
First PR
I fix it, but I don't know how to continue :sweat_smile:.

Reply:
Question, do you have to split arguments up into an array? The `spawn` docs show examples where they have multiple command parameter in a single string.

Reply:
Looking at this again.

`Kernel.spawn`: This is to avoid creating a sub-process, right?

`arguments.gsub("\"","").split(" ")` I still don't fully understand what this is solving. The docs seem to indicate that `spawn` can take a single string with arguments just like `system`. So what is this line doing?